cli/utils.go

```diff
@@ -162,8 +177,9 @@ func FilterNotCheckpointedOrWithdrawnValidators(
 
 		notCheckpointed := validatorInfo.LastCheckpointedAt != lastCheckpoint
 		notWithdrawn := validatorInfo.Status != ValidatorStatusWithdrawn
+		notInactive := validatorInfo.Status != ValidatorStatusInactive
 
-		if notCheckpointed && notWithdrawn {
+		if notCheckpointed && notWithdrawn && notInactive {
```

Comment:
do we want to just explicitly check `validatorInfo.Status == ValidatorStatusActive` instead?

Reply:
You could combine the last 2 into a `isActive` flag if you want but idc personally

Reply:
jk this is nicer i change mind